modules/release-notes/pages/relnotes.adoc

```diff
@@ -1,6 +1,236 @@
++
+{supported-platforms-url}
+
+* After 7.1.2, offline upgrade for Windows will not be supported.
+Note that this means that no procedure will be available for the upgrade of a single-node Windows cluster.
++
+Install, uninstall, and repair for Windows will continue to be supported after 7.1.2.
+
+=== Fixed Issues
+
+This release contains the fixes listed below.
+
+=== Data Service
+
+[#table-known-issues-712-dataservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-53046[MB-53046^]
+| wait for seqno persistence won't timeout on idle vbucket
+
+| https://issues.couchbase.com/browse/MB-51608[MB-51608^]
+| Memcached crashes in 20 bucket throughput test due to exception
+
+| https://issues.couchbase.com/browse/MB-47267[MB-47267^]
+| Vbucket stats call to KV can timeout during delta node recovery preparation
+
+|===
+
+=== Query Service
+
+[#table-known-issues-712-queryservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52254[MB-52254^]
+| Improve pagination queries with fetch
+
+| https://issues.couchbase.com/browse/MB-52764[MB-52764^]
+| Race condition between stop signal and timeout
+
+| https://issues.couchbase.com/browse/MB-52253[MB-52253^]
+| Push filters to index scan to potentially reduce fetch size
+
+| https://issues.couchbase.com/browse/MB-52959[MB-52959^]
+| UNNEST query 'usedMemory' issue when using Query Memory Quota
+
+| https://issues.couchbase.com/browse/MB-52443[MB-52443^]
+| Include missing key Index syntax on Index workbench doesn't show include keyword in definition
+
+| https://issues.couchbase.com/browse/MB-31640[MB-31640^]
+| subqueries should be advised, explained and monitored
+
+|===
+
+=== Index Service
+
+[#table-known-issues-712-indexservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52063[MB-52063^]
+| With Collection Indexer should index leading MISSING entries
+
+| https://issues.couchbase.com/browse/MB-51947[MB-51947^]
+| indexer blocked during storage warmup on MOI storage, causing rebalance failure.
+
+| https://issues.couchbase.com/browse/MB-52443[MB-52443^]
+| Include missing key Index syntax on Index workbench doesn't show include keyword in definition
+
+|===
+
+=== Search Service
+
+[#table-known-issues-712-searchservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-51760[MB-51760^]
+| GeoJSON Spatial Indexing support
+
+|===
+
+=== Analytics Service
+
+[#table-known-issues-712-analyticsservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52783[MB-52783^]
+| Select * query throws Failure contacting server for parquet files
+
+|===
+
+
+=== Eventing Service
+
+[#table-known-issues-712-eventingservice, cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-52376[MB-52376^]
+| CI test for eventing multi collection
```

Comment:
@tonyjhillman This is an internal test - don't think customer need to see this MB

Reply:
Removed.